src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,13 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <>
+    {todos.map(todo => (
+      <section className="TodoList" key={todo.id}>
```

Comment:
Each todo item is wrapped in a section with the same className 'TodoList'. It would make more sense to wrap the entire list in this section instead of each individual item. Consider moving this section tag outside the map function.